doc/nrf/includes/boardname_tables/sample_boardnames.txt

```diff
@@ -1,7 +1,7 @@
 .. _sampleboardnames:
 
 Set used by BLE (Central BAS, Central DFU SMP, Central HIDS, Central UART, EnOcean, Mesh Light, Mesh Light Switch, Peripheral HIDS Keyboard,  Peripheral HIDS mouse,
-Peripheral UART, NUS Shell Transport, Throughput) and NFC samples
+Peripheral UART, NUS Shell Transport, Throughput), Radio Test and NFC samples
```

Comment:
Do we need to keep this as an up to date list of all the samples that use it? This seems like a maintance problem.

Reply:
@ru-fu @b-gent @umapraseeda 
I have seen multiple PRs that need to update this line now. Can we make this easier for us?
It would be better to describe the set, instead of listing samples that use it.

Reply:
This line will be updated again in an upcoming PR, where the supported boards for the rest of the Bluetooth samples will be updated. For now, removed the listing of samples and shortened it to Bluetooth LE samples wherever applicable.